### ISSUE TITLE
New Release?

### ISSUE THREAD

**Issue by bexelbie** [linkcheck/linkchecker#81]
Should we cut a new release to add in the current fixes?  even 9.3.2

**Comment by mgedmin** [comment]
Duplicate of #4.  We're blocked by two things:

- not having PyPI access to the original linkchecker project
- not having active volunteers who would actually do the release
